lib/l10n_utils/tests/test_template.py

```diff
@@ -125,3 +125,37 @@ def test_render_no_locale(self, django_render, get_lang_path):
         # Note: no .locale on request
         # Should not cause an exception
         render(request, '500.html')
+
+
+@patch.object(env, 'loader', FileSystemLoader(TEMPLATE_DIRS))
+@override_settings(ROOT=ROOT)
+class TestLocaleTemplates(TestCase):
+    def setUp(self):
+        self.client = Client()
+
+    def test_default_template(self):
+        '''
+        No custom templates exist for en-US or de. Should load
+        test_files/templates/firefox/new.html.
+        '''
+        response = self.client.get('/en-US/firefox/new/')
+        self.assertTemplateUsed(response, 'firefox/new.html')
+
+        response = self.client.get('/de/firefox/new/')
+        self.assertTemplateUsed(response, 'firefox/new.html')
+
+    def test_bedrock_locale_template(self):
```

Comment:
Another way to do these tests if this isn't working (and I don't see any other use of `assertTemplateUsed` in bedrock) is to use mock:

```python
from django.template import TemplateDoesNotExist
from django.test import RequestFactory, override_settings

from mock import ANY, patch

from lib.l10n_utils import render


@patch.object(env, 'loader', FileSystemLoader(TEMPLATE_DIRS))
@override_settings(ROOT=ROOT)
class TestLocaleTemplates(TestCase):
    def setUp(self):
        self.rf = RequestFactory()

    @patch('lib.l10n_utils.django_render')
    def test_l10n_render(self, mock_dj_render):
        # using a list here means that the first call will raise the exception, and 2nd call
        # will return True. You could add another exception class to the list and it would mean
        # the 2nd call would also raise if you wanted to test it falling all the way through.
        mock_dj_render.side_effect = [TemplateDoesNotExist, True]
        request = self.rf.get('/')
        request.locale = 'de'
        render(request, 'firefox/new.html')
        mock_dj_render.assert_called_with(request, 'firefox/new.html', ANY)
```
I realize this seems a bit crazier, but it is more of a unit test and should be faster and you don't really need those test files if this works.